set.mm

```diff
@@ -735799,6 +735836,340 @@ their group (addition) operations are equal for all pairs of elements of
+       (infinite) product in [Lang] p. 5, two last formulas.  (Contributed by
+       AV, 27-Dec-2023.) $)
+    gsumfsupp $p |- ( ph -> ( G gsum ( F |` I ) ) = ( G gsum F ) ) $=
+      ( csupp co wss eqimss2i a1i gsumres ) ABCDEGFHIJLMNDHPQZFRAFUBKSTOUA $.
+  $}
+
+  ${
+    $d A j k $.  $d B j k $.  $d C j k $.  $d G j k $.  $d F j k $.  $d V j $.
+    $d W k $.  $d .0. j k $.  $d ph j k $.
+    gsumxp2.b $e |- B = ( Base ` G ) $.
+    gsumxp2.z $e |- .0. = ( 0g ` G ) $.
+    gsumxp2.g $e |- ( ph -> G e. CMnd ) $.
+    gsumxp2.a $e |- ( ph -> A e. V ) $.
+    gsumxp2.r $e |- ( ph -> C e. W ) $.
+    gsumxp2.f $e |- ( ph -> F : ( A X. C ) --> B ) $.
+    gsumxp2.w $e |- ( ph -> F finSupp .0. ) $.
+    $( Write a group sum over a cartesian product as a double sum in two ways.
+       This corresponds to the first equation in [Lang] p. 6.  (Contributed by
+       AV, 27-Dec-2023.) $)
+    gsumxp2 $p |- ( ph
+          -> ( G gsum ( k e. C |-> ( G gsum ( j e. A |-> ( j F k ) ) ) ) )
+           = ( G gsum ( j e. A |-> ( G gsum ( k e. C |-> ( j F k ) ) ) ) ) ) $=
+      ( co wcel cv cmpt cgsu csupp fovrnda fsuppimpd wa wbr wn wceq cop cfv cxp
+      simpl opelxpi ad2antlr simpr eldifd cvv ssidd xpexd c0g fvexi a1i suppssr
+      cdif syl2an2r ex df-br notbii df-ov eqeq1i 3imtr4g impr gsumcom3 eqcomd )
+      AHEBHFDEUAZFUAZGSZUBUCSUBUCSHFDHEBVSUBUCSUBUCSABCDGKUDSZEFHIJVSKLMNOPAVQV
+      RCBDGQUEAGKRUFAVQBTVRDTUGZVQVRVTUHZUIZVSKUJZAWAUGZVQVRUKZVTTZUIZWFGULZKUJ
+      ZWCWDWEWHWJWEAWHWFBDUMZVTVFTWJAWAUNWEWHUGWFWKVTWAWFWKTAWHVQVRBDUOUPWEWHUQ
+      URAWKCUSGUSVTWFKQAVTUTABDIJOPVAKUSTAKHVBMVCVDVEVGVHWBWGVQVRVTVIVJVSWIKVQV
+      RGVKVLVMVNVOVP $.
+  $}
+
+
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Group multiple operation (extension) and cyclic monoids
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  ${
+    $d B i x $.  $d F i $.  $d N i x $.  $d X i x $.
+    mulgnngsum.b $e |- B = ( Base ` G ) $.
+    mulgnngsum.t $e |- .x. = ( .g ` G ) $.
+    mulgnngsum.f $e |- F = ( x e. ( 1 ... N ) |-> X ) $.
+    $( Group multiple (exponentiation) operation at a positive integer
+       expressed by a group sum.  (Contributed by AV, 28-Dec-2023.) $)
+    mulgnngsum $p |- ( ( N e. NN /\ X e. B ) -> ( N .x. X ) = ( G gsum F ) ) $=
+      ( vi cn wcel wa cfv c1 cseq co adantr cv cplusg csn cxp cuz elnnuz biimpi
+      cgsu cfz cmpt wceq a1i eqidd simpr fvmptd elfznn fvconst2g syl2an seqfveq
+      weq eqtr4d cvv eqid elfvex eleq2s adantl fmptd gsumval2 mulgnn 3eqtr4rd
+      cbs ) FLMZGBMZNZFEUAOZDPQOFVNLGUBUCZPQZOEDUGRFGCRVMVNKDVOPFVKFPUDOMZVLVKV
+      QFUEUFSZVMKTZPFUHRZMZNZVSDOGVSVOOZWBAVSGGVTDBDAVTGUIUJWBJUKWBAKUSNGULVMWA
+      UMVMVLWAVKVLUMZSUNVMVLVSLMWCGUJWAWDVSFUOLGVSBUPUQUTURVMBVNDEPFVAHVNVBZVLE
+      VAMZVKWFGEVJOBGEVJVCHVDVEVRVMAVTGBDVMVLATVTMWDSJVFVGBVNVPCEFGHWEIVPVBVHVI
+      $.
+
+    $( Group multiple (exponentiation) operation at a nonnegative integer
```

Comment:
It depends whether the law is written additively of multiplicatively. Probably "iterated group law". Then, you can add: "When the law is written multiplicatively, the iterated group law is also called exponentiation." (I realize these changes should happen at other places already in the main section. This can be done later.)

Reply:
"Group multiple (exponentiation)" is the term currently used in set.mm, so I will keep it here for the moment.